proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -221,3 +221,24 @@ message Tip {
   // tipper is the address of the account paying for the tip
   string tipper = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
 }
+
+// AuxSignerData is the intermediary format that an auxiliary signer (e.g. a
+// tipper) builds and sends to the fee payer (who will build and broadcast the
+// actual tx). AuxSignerData is not a valid tx in itself, and will be rejected
+// by the node if sent directly as-is.
+//
+// Since: cosmos-sdk 0.45
+message AuxSignerData {
+  // address is the bech32-encoded address of the auxiliary signer. If using
+  // AuxSignerData across different chains, the bech32 prefix of the target
+  // chain (where the final transaction is broadcasted) should be used.
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+	// sign_doc is the SIGN_MOD_DIRECT_AUX sign doc that the auxiliary signer
+  // signs. Note: we use the same sign doc even if we're signing with
+  // LEGACY_AMINO_JSON.
+	SignDocDirectAux sign_doc = 2;
```

Comment:
The Tx body_bytes is already in SignDocDirectAux. However, do we also want to copy it here as a top-level `TxBody body = 3;` field?

Pros:
- If using JSON, then the fee payer can have human readable description of msgs, without needing to. Currently, we only see base64 bytes.

Cons:
- More data to be transmitted over the wire (as it's a duplicate of `sign_doc.body_bytes`)
- only useful for JSON to be read by humans. In general, for machine-to-machine, proto binary should be preferred (more compact)